sbncode/CAFMaker/CMakeLists.txt

```diff
@@ -39,9 +39,11 @@ art_make_library( LIBRARY_NAME sbncode_CAFMaker
                   sbnobj::Common_Trigger
                   sbnobj::SBND_CRT
                   sbnobj::SBND_Timing
+		  sbndobj_BlipDataTypes
```

Comment:
Please fix the alignment. My suggestion is to copy the line above, to make sure the same indentation characters are use.